.github/workflows/weekly-r-cmd-check.yaml

```
@@ -0,0 +1,54 @@
+
+    env:
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
+      R_KEEP_PKG_SOURCE: yes
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: r-lib/actions/setup-pandoc@v2
+
+      - name: Setup .NET
+        uses: actions/setup-dotnet@v1
```

actionlint (1.7.12)

[error] the runner of "actions/setup-dotnet@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)